pkg/db/remoteRequest.go

```diff
@@ -72,7 +72,7 @@ func createRequest(dbKeys []string, values [][]byte, signer RequestSigner) (*req
 	}
 	if sig == nil {
 		log.Error("Signature was not generated")
-		return nil, errors.Errorf("Could not generate a signature for  hash: %v", hash)
+		return nil, errors.Wrapf(err, "Could not generate a signature for  hash: %v", hash)
```

Comment:
```suggestion
		return nil, errors.Wrapf(err, "generate a signature for  hash: %v", hash)
```
we don't need words like this, just need to describe when the error occurred. It is obvious that there is `couldn't` :smile: 

Also, all errors are lower case and no full stop at the end.